web/content/js/versions.js

```diff
@@ -167,6 +167,86 @@ const navVersions = [
       }
     ]
   },
+  {
+    title: "Foreman 3.4 - Katello 4.6 (supported)",
```

Comment:
Do we have to set 3.2 to unsupported now?

Reply:
No, only once Foreman 3.4 is released (GA, General Availability is also sometimes used).

Reply:
Okay, understood.